.kokoro/runintegrationtests.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e
+
+cd ..
+
+# Build the libraries w/o running unit tests.
```

Comment:
Maybe we should run the unit tests as well? Make this more of a "full build".

Reply:
Can't hurt, updated.